contentcuration/contentcuration/frontend/shared/views/form/ContentDefaults.vue

```diff
@@ -0,0 +1,230 @@
+        type: Object,
+        default() {
+          return { ...ContentDefaultsDefaults };
+        },
+      },
+      defaultsTitle: {
+        type: String,
+      },
+      defaultsSubTitle: {
+        type: String,
+      },
+      thumbnailsSubTitle: {
+        type: String,
+      },
+    },
+    data() {
+      const normalized = normalizeContentDefaults(this.contentDefaults);
+
+      return {
+        author: normalized.author,
+        provider: normalized.provider,
+        aggregator: normalized.aggregator,
+        copyrightHolder: normalized.copyrightHolder,
+        license: findLicense(normalized.license, { license_name: '' }).license_name,
+        licenseDescription: normalized.licenseDescription,
+        autoDeriveAudioThumbnail: normalized.autoDeriveAudioThumbnail,
+        autoDeriveDocumentThumbnail: normalized.autoDeriveDocumentThumbnail,
+        autoDeriveHtml5Thumbnail: normalized.autoDeriveHtml5Thumbnail,
+        autoDeriveVideoThumbnail: normalized.autoDeriveVideoThumbnail,
+      };
+    },
+    computed: {
+      licenseOpts() {
+        const licenseOpts = Constants.Licenses.map(license => ({
+          value: license.license_name,
+          text: this.translateConstant(license.license_name),
+        }));
+
+        licenseOpts.unshift({
+          value: '',
+          text: this.$tr('noLicense'),
+        });
+        return licenseOpts;
+      },
+      isCustomLicense() {
+        return findLicense(this.license, { is_custom: false }).is_custom;
+      },
+    },
+    methods: {
+      emitChange() {
+        // When any field in our component changes, this gets triggered which then triggers the
+        // event that updates the prop passed into us as `contentDefaults` in the parent component.
+        // This assigns the data into a object, with snake cased keys instead of camel cased here
+        this.$nextTick(() => {
+          this.$emit(
+            'change',
+            Object.entries(ContentDefaults).reduce((data, [snakeCaseName, camelCaseName]) => {
+              const value = this[camelCaseName] === '' ? null : this[camelCaseName];
+              return Object.assign(data, { [snakeCaseName]: value });
+            }, {})
+          );
+        });
+      },
+    },
+    $trs: {
+      defaultsTitle: 'New resource defaults (optional)',
+      defaultsSubTitle: 'New resources you create will be automatically given these fields',
+      author: 'Author',
+      authorPlaceholder: 'Enter author name...',
+      aggregator: 'Aggregator',
+      aggregatorPlaceholder: 'Enter aggregator name...',
+      provider: 'Provider',
+      providerPlaceholder: 'Enter provider name...',
+      license: 'License',
+      licenseDescription: 'License description',
+      licenseDescriptionPlaceholder: 'Enter license description...',
+      noLicense: 'No license selected',
+      copyrightHolder: 'Copyright Holder',
+      copyrightHolderPlaceholder: 'Enter copyright holder name...',
```

Comment:
Our convention in Kolibri, which I believe we'll want to carry over to Studio, is that we _only_ use labels - not placeholders also.

Some references:

* [Kolibri guidelines](https://docs.google.com/document/d/1iMh4Q43O2eGDRqFeGZeoysMyaJ61TeOrPC7n1mm2fZ0/edit)
* Issue for an upcoming upgrade: https://github.com/learningequality/kolibri/issues/6033 and a [related recent slack thread](https://learningequality.slack.com/archives/C0LKG14NL/p1573139501088800)

Let's try to use these conventions so we don't have a lot of re-work later

Reply:
I kept the existing approach of the Vue form, which already had placeholders, and carried the placeholders over from the existing form. The design does not have placeholders, so I'll just remove them all.